CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to DBOS-TS
+
+Thank you for considering contributing to DBOS Typescript SDK. We welcome contributions from everyone, whether it's bug fixes, feature enhancements, documentation improvements, or any other form of contribution.
+
+## How to Contribute
+
+To get started with DBOS-TS, please read the [README](README.md).
+
+You can contribute in many ways. Some simple ways are:
+* Use the SDK and open issues to report any bugs, questions, concern with the SDK, samples or documentation.
+* Respond to issues with advice or suggestions.
+* Participate in discussions in our [Discord](https://discord.gg/fMwQjeW5zg) channel.
+* Contribute fixes and improvement to code, samples or documentation.
+
+### To contribute code, please follow these steps:
+
+1. Fork the the github repository to your own account.
+
+2. Clone the forked repository to your local machine.
+
+3. Create a branch.
+
+4. Make the necessary change to code, samples or documentation.
+
+5. Write tests.
+
+6. Commit the changes to your forked repository.
+
+7. Submit a pull request to DBOS repository.
+
+## Requesting features
+
+If you have a feature request or an idea for an enhancement, feel free to open an issue on GitHub. Describe the feature or enhancement you'd like to see and why it would be valuable. Discuss it with the community on the [Discord](https://discord.gg/fMwQjeW5zg) channel.
+
+## Discuss with the community
+
+If you are stuck, need help, or wondering if a certain contribution will be welcome, please ask! Either in our [Discord](https://discord.gg/fMwQjeW5zg) or Github discussions.
```

Comment:
Either in our... -> Reach out to us in [Discord](https://discord.gg/fMwQjeW5zg) or start a GitHub discussion.